### PR TITLE
[CM_APP_56] Gayatri | Add swagger changes for verifying user for recover cmid

### PR DESCRIPTION
**Pull request by gayatrijagtap**
…d patient/recovery/verifyotp

### CODE REVIEW

File: src/main/resources/static/api.yaml [hunk 1]
```diff
@@ -1385,6 +1396,65 @@ paths:
...
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/VerifyOtpRequest'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/VerifyOtpRequest'
+      responses:
+        '200':
```
Comment (angshu) [inline src/main/resources/static/api.yaml:1415]: What would the responsebody be? How do you inform the patient of the recovered CM ID? Are you supposed to send over SMS or display over screen?

File: src/main/resources/static/api.yaml [hunk 2]
```diff
@@ -1385,6 +1396,65 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /patients/recovery/verifyotp:
```
Comment (angshu) [inline src/main/resources/static/api.yaml:1399]: Can you think of better name? /patients/profile/recovery-confirm

File: src/main/resources/static/api.yaml [hunk 3]
```diff
@@ -1342,14 +1342,14 @@ paths:
               $ref: '#/components/schemas/RecoverCmIdRequest'
       responses:
         '200':
-          description: Found a patient
+          description: Details of the session and otp are sent in response. otpMedium defines verified identifier where it would be sent to. Its value is mentioned in otpMediumValue. OTP has an expiry time in minutes which is also sent.
```
Comment (angshu) [inline src/main/resources/static/api.yaml:1345]: please rename the endpoint as "/patients/profile/recovery-init"

File: src/main/resources/static/api.yaml (hunk 1 above)
Comment (gayatrijagtap) [reply src/main/resources/static/api.yaml:1415]: the response body is there which is sending cm id

File: src/main/resources/static/api.yaml (hunk 2 above)
Comment (gayatrijagtap) [reply src/main/resources/static/api.yaml:1399]: will rename it

File: src/main/resources/static/api.yaml (hunk 3 above)
Comment (gayatrijagtap) [reply src/main/resources/static/api.yaml:1345]: yeah will do

**Pull request merged**